src/components/taskDetails/index.tsx

```diff
@@ -304,15 +312,23 @@ const TaskDetails: FC<Props> = ({ taskID }) => {
                             >
                                 <Details
                                     detailType={'Started On'}
-                                    value={convertTimeStamp(
-                                        taskDetailsData?.startedOn ?? 0
-                                    )}
+                                    value={
+                                        taskDetailsData?.startedOn
+                                            ? convertTimeStamp(
+                                                  taskDetailsData.startedOn
+                                              )
+                                            : 'N/A'
+                                    }
                                 />
                                 <Details
                                     detailType={'Ends On'}
-                                    value={convertTimeStamp(
-                                        taskDetailsData?.endsOn ?? 0
-                                    )}
+                                    value={
+                                        taskDetailsData?.endsOn
+                                            ? convertTimeStamp(
+                                                  taskDetailsData.endsOn
+                                              )
+                                            : 'TBD'
+                                    }
```

Comment:
here also

Reply:
https://github.com/Real-Dev-Squad/website-status/pull/900/commits/13c453d481573adcfcc2c069cbbcdd3251addba9#diff-b3db363b4a8fa025fc84f53f01361afda07866b41ca4b4a28baede8b2a9cd844L96 do you want me to use it like this?